### PR TITLE
How we turned a monolitic UI into multiple modular applications

### PR DESCRIPTION
**Pull request by thom4parisot**
Hello,

thanks for organising the event :-)

Greetings from London!

Abstract:

---

React.js is **not** cool.
Spending efforts to convince people wary about the hype is not cool either.
Spending less time chasing bugs **is** cool.

I spent 3 afternoons to break up an existing Web application into npm packages and demoed our visual atoms as an autonomous living styleguide.
When I have been asked how I achieved this so quickly: I answered I created and assembled React.js components.

Developing new features became faster than one would expect.
The time spent in code reviews and maintenance time diminished as well.
And even better: now, our UI designers test and prototype their ideas right into the components. They only have to type `npm start` to see their work reflected in their Web browser.

This talk relates this migration journey:
- breaking up a common problem through incremental improvement
- elevating the browserify pipeline to a first class citizen
- making our interface and components reusable by default
- building and integrating our shiny components in non-browserified applications

Basically, this is a story on how I turned _frustrated_ developers into _happy_ composers pairing with UI designers.

### CODE REVIEW

**Comment by gabssnake** [comment]
:+1:

**Comment by patcito** [comment]
Thank you for taking your time and submitting your proposal. We have received close to a 100 and had to select only a dozen of them. It wasn’t an easy task and we're sorry that yours didn't make the cut. If you are still interested in coming and talk, there will be a lightning talk session during the conf in front of the whole audience with a great prize for the best talk, we will post more details soon. Cheers, Patrick.
